web/src/stream_popover.js

```diff
@@ -245,9 +248,12 @@ function build_topic_popover(opts) {
     const content = render_topic_sidebar_actions({
         stream_name: sub.name,
         stream_id: sub.stream_id,
+        stream_muted: sub.is_muted,
         topic_name,
         topic_muted,
+        topic_unmuted,
         can_move_topic,
+        dev_env: page_params.development_environment,
```

Comment:
Call this `development_environment`; we use that name in many templates and it makes grepping way more convenient.

(In general, one should try to avoid renaming a property to a slightly different name like this when passing it between different parts of the system -- it's almost always bad for readability and has no real benefit).